summingbird-storm/src/main/scala/com/twitter/summingbird/storm/FlatMapBoltProvider.scala

```diff
@@ -79,10 +82,10 @@ case class FlatMapBoltProvider(builder: StormTopologyBuilder, node: StormNode) {
   private val maxEmitPerExecute = getOrElse(DEFAULT_MAX_EMIT_PER_EXECUTE)
   logger.info(s"[$nodeName] maxEmitPerExecute : ${maxEmitPerExecute.get}")
 
-  private val usePreferLocalDependency = getOrElse(DEFAULT_FM_PREFER_LOCAL_DEPENDENCY)
-  logger.info(s"[$nodeName] usePreferLocalDependency: ${usePreferLocalDependency.get}")
```

Comment:
It's here now https://github.com/twitter/summingbird/pull/731/files#diff-ada7a53f54309669e9f2c41907787a5cR48